mindmeld/components/_config.py

```diff
@@ -508,7 +508,7 @@ def is_duckling_configured(app_path):
 
 def get_system_entity_url_config(app_path):
     if not app_path:
-        return DEFAULT_DUCKLING_URL
+        return os.environ.get('MM_DUCKLING_URL', DEFAULT_DUCKLING_URL)
```

Comment:
Is this better than setting it from within the config file? e.g.
```
NLP_CONFIG = {
    'system_entity_recognizer': {'type': 'duckling', 'url': os.environ.get('MM_DUCKLING_URL')}
}

Reply:
I am thinking of cases where you want to call the numerical parsing service independently from the app

Reply:
But it is called from the app here as well.

Reply:
I just looked for how to do this, sad that you cant. Only way to set up mind meld on a local machine using docker compose seems to be with network mode host?